### PR TITLE
Updated to 2.7.0

### PR DESCRIPTION
**Pull request by npanpaliya**
## Checklist before submitting

- [x] Did you read the [contributor guide](https://github.com/open-ce/open-ce/blob/main/CONTRIBUTING.md)?
- [ ] Did you update any affected [documentation](https://github.com/open-ce/open-ce/blob/main/doc/)?
- [ ] Did you write any tests to validate this change?

## Description

Updates to 2.7.0

## Review process to land 

1. All tests and other checks must succeed.
2. At least one [maintainer](https://github.com/open-ce/open-ce/blob/main/MAINTAINERS.md) must review and approve.
3. If any  [maintainer](https://github.com/open-ce/open-ce/blob/main/MAINTAINERS.md) requests changes, they must be addressed.

### CODE REVIEW

**Review by alexlang74 — COMMENTED** [review]
Why did you change `{{ version }}` to a "hardcoded" `2.7.0` ?

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
+{% set version = "2.7.0" %}
 
 package:
   name: keras
   version: {{ version }}
 
 source:
-  url: https://files.pythonhosted.org/packages/5a/38/04d9b7fb53acdf861df2c4505fa96b06c779817a511e94b8882d284ba360/keras-{{ version }}-py2.py3-none-any.whl
-  sha256: 504af5656a9829fe803ce48a8580ef16916e89906aceddad9e098614269437e7
+  url: https://files.pythonhosted.org/packages/6b/8b/065f94ba03282fa41b2d76942b87a180a9913312c4611ea7d6508fbbc114/keras-2.7.0-py2.py3-none-any.whl
```
Comment (cdeepali) [inline recipe/meta.yaml]: Did we hardcode version in the URL due to any issue?

Comment (npanpaliya) [reply recipe/meta.yaml]: It was hard-coded for some debugging purpose when keras 2.7 was not out. I'll revert that.

**Comment by npanpaliya** [comment]
> Why did you change `{{ version }}` to a "hardcoded" `2.7.0` ?

Yes, it was unintentional. I changed it.

**Review by alexlang74 — APPROVED** [review]

**Review by cdeepali — APPROVED** [review]

**Pull request merged**